connector/elasticapmconnector/config.go

```diff
@@ -64,6 +65,18 @@ type AggregationConfig struct {
 	// in all other cases -- using this configuration may lead to invalid behavior,
 	// and will not be supported.
 	Intervals []time.Duration `mapstructure:"intervals"`
+
+	// ResourceLimit defines the max cardinality of resources
+	ResourceLimit lsmconfig.LimitConfig `mapstructure:"resource_limit"`
+
+	// ScopeLimit defines the max cardinality of scopes within a resource
+	ScopeLimit lsmconfig.LimitConfig `mapstructure:"scope_limit"`
+
+	// MetricLimit defines the max cardinality of metrics within a scope
+	MetricLimit lsmconfig.LimitConfig `mapstructure:"metric_limit"`
+
+	// DatapointLimit defines the max cardinality of datapoints within a metric
+	DatapointLimit lsmconfig.LimitConfig `mapstructure:"datapoint_limit"`
```

Comment:
These configs are too generic for `elasticapmconnector`. I would propose to have configurations for `service_summary`, `service_transaction`, `transaction`, and `service_destination` and using them to configure the correct lsm interval configurations rather than directly exposing lsminterval configs. Again, https://github.com/elastic/opentelemetry-collector-components/issues/141#issuecomment-2592951016 might help a bit in understanding how we can do this.

Reply:
Hmm its not clear to me how we can map configurations for `service_summary`, `service_transaction`, `transaction`, and `service_destination` to lsminterval configs. To me it seems that these would apply to the `resource_limit` only. I am most likely missing something. I will spend more time understanding the  `elasticapmconnector` metrics and how they affect the different levels of cardinality in the `lsmintervalprocessor`.